### PR TITLE
Add docker file and the necessary documentation

### PR DESCRIPTION
**Pull request by fai555**

### CODE REVIEW

File: docs/docker_prediction_request_client.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+# relative path to the output json. i.e. config/out_image1.json. The path is relative to VOLUME_PATH
+export OUTPUT_JSON=""
+# relative path to the label_map.pbtxt. i.e. config/label_map.pbtxt. The path is relative to VOLUME_PATH
+export LABEL_MAP=""
+# Specify True if you want to save the output image. 
+export SAVE_OUTPUT_IMAGE=True
+# You can use the publicly available docker image or use your own image
+export DOCKER_IMAGE_NAME="asia.gcr.io/im-mlpipeline/tensorflow-serving-sidecar-client:latest"
+
+docker run -it -v ${VOLUME_PATH}:/app/tensorflow-serving_sidecar/config -t ${DOCKER_IMAGE_NAME} client.py --server_url=${SERVER_URL}  --image_path=${IMAGE_PATH} --output_json=${OUTPUT_JSON} --save_output_image=${SAVE_OUTPUT_IMAGE} --label_map=${LABEL_MAP}
```
Comment (innovolt) [inline docs/docker_prediction_request_client.md:37]: Could you please tell how to get the saved output image in host filesystem? I see that TF Serving says the output image has been saved successfully but I don't see it on my host filesystem even though the host dir (i.e. VOLUME_PATH) is mounted in docker container filesystem.

File: docs/docker_prediction_request_client.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# Build Docker Images
+
+You can build your own docker image using the Dockerfile in the docker direcoty.
+```
+docker build -t tensorflow-serving-sidecar-client:latest .
```
Comment (navlok-mishra) [inline docs/docker_prediction_request_client.md:5]: Getting the following error:

W: The repository 'http://ppa.launchpad.net/jonathonf/python-3.6/ubuntu xenial Release' does not have a Release file.
E: Failed to fetch http://ppa.launchpad.net/jonathonf/python-3.6/ubuntu/dists/xenial/main/binary-amd64/Packages  403  Forbidden
E: Some index files failed to download. They have been ignored, or old ones used instead.
The command '/bin/sh -c apt-get update' returned a non-zero code: 100

Comment (fai555) [reply docs/docker_prediction_request_client.md:5]: **ppa:jonathonf was removed by the developer.**
https://launchpad.net/~jonathonf

You can refer to this discussion in the following link.
https://gist.github.com/monkut/c4c07059444fd06f3f8661e13ccac619#gistcomment-3112630


I have updated the Dockerfile and tested it. It's working now.

Comment (fpaupier) [reply docs/docker_prediction_request_client.md:5]: Thanks for the update @fai555 .

**Pull request merged**